website/prh.yml

```
@@ -0,0 +1,5 @@
+# prh version
+version: 1
+rules:
+  - expected: lint
+    patterns: リント
```

We should add more patterns, but 
```suggestion
  - expected: lint
    patterns: リント
  - expected: linter
    patterns: 
      - リンター
      - リンタ  
  - expected: formatter
    patterns:
      - フォーマッター
      - フォーマッタ
```